pkg/check/smoke/smoke.go

```diff
@@ -9,26 +9,56 @@ import (
 
 	"github.com/ethersphere/beekeeper/pkg/bee"
 	"github.com/ethersphere/beekeeper/pkg/beeclient/api"
+	"github.com/ethersphere/beekeeper/pkg/beekeeper"
 	"github.com/ethersphere/beekeeper/pkg/random"
 )
 
+// TODO: remove need for node group, use whole cluster instead
```

Comment:
not sure if this TODO is really relevant, since the test takes a cluster as an argument. Maybe instead, we can just have a wildcard possibility so that if you select `*` as a nodegroup you test against the whole cluster, but actually the test in its current form can be preserved, not too bad to have the possibility of testing over one node group

Reply:
Good idea, lets discuss this on planning